src/platform-includes/configuration/integrations/default-integrations/node.mdx

```diff
@@ -0,0 +1,184 @@
+System integrations are enabled by default to integrate into the standard library or the interpreter itself. They are documented so you can be both aware of what they do and disable them if they cause issues.
+
+## Enabled by Default
+
+### InboundFilters
+
+_Import name: `Sentry.Integrations.InboundFilters`_
+
+This integration allows developers to ignore specific errors based on the error message or the URL from which the exception originated.
+
+To configure it, use the `ignoreErrors`, `denyUrls`, and `allowUrls` SDK options directly.
+(Keep in mind that `denyUrls` and `allowUrls` only work for captured exceptions, not raw message events.)
+
+### FunctionToString
+
+_Import name: `Sentry.Integrations.FunctionToString`_
+
+This integration allows the SDK to provide original function and method names, even when those functions or methods
+are wrapped by our error or breadcrumb handlers.
+
+## Node-specific
+
+### Console
+
+_Import name: `Sentry.Integrations.Console`_
+
+This integration wraps the `console` module in order to record all of its calls as breadcrumbs.
+
+### Http
+
+_Import name: `Sentry.Integrations.Http`_
+
+This integration wraps `http` and `https` modules to capture all network requests as breadcrumbs and/or tracing spans.
+
+Available options:
+
+```javascript
+{
+  breadcrumbs: boolean; // default: true
+  tracing: boolean; // default: false
+}
+```
+
+### OnUncaughtException
+
+_Import name: `Sentry.Integrations.OnUncaughtException`_
+
+This integration attaches a global uncaught exception handler. It can be modified to provide a custom shutdown function.
+The `onFatalError` option is meant to perform a cleanup before the process exits, not fully prevent it from exiting.
+
+<Alert level="warning" title="Note">
```

Comment:
Does this need to be a warning? Are you breaking something? If not, let's bump this down to a regular note.
```suggestion
<Alert level="warning">
```